src/Resolver.js

```diff
@@ -9,6 +9,19 @@ class Resolver {
   constructor(options = {}) {
     this.options = options;
     this.cache = new Map();
+    this.pathIndicators = {
+      // resolve '~/' to root path of project
+      '~/': path.dirname(this.options.cacheDir)
+    };
+  }
+
+  resolvePath(filename) {
+    for (let [indicator, exactPath] of Object.entries(this.pathIndicators)) {
+      if (filename.startsWith(indicator)) {
+        return path.resolve(exactPath, filename.slice(indicator.length));
```

Comment:
Will this also handle urlImports in for example html?

Reply:
it cann't handle in html which is entry